### DISCUSSION TITLE
Can I stake IOTA for Assembly

### DISCUSSION THREAD

**Discussion by MELASWOJTE** [iotaledger/firefly#6285] [Help & Support]
Can i stake from now my IOTA tokens to get ASMB. Can I start now or is it too late?

I am asking because in the Firefly version 1.7.0 or 1.7.1 i dont see how to do it?

**Comment by HankMoody089** [comment]
Yes you can. Go to the settings of Firefly and choose automatic node selection and then you should find the staking event.

**Reply by MELASWOJTE** [reply]
Firefly 1.7.0 or 1.7.1?

**Reply by HankMoody089** [reply]
Always use the latest version, so 1.7.1

**Reply by MELASWOJTE** [reply]
ok, but i have there only the shimmer.network node. As well i can only see in the wallet only the shimmer tokens.
In the node selection i can not choose between manual and and automatic, like in the version 1.7.0.

In 1.7.1 under node configuration i have: manage the list of nodes you connect to.

**Comment by HankMoody089** [comment]
This is from my 1.7.1 Firefly version. There must be a automatic setting in node selection.


<img width="573" alt="Bildschirmfoto 2023-03-28 um 16 32 22" src="https://user-images.githubusercontent.com/60474745/228272382-807af0ef-6a1c-4e00-997f-e7c48531cab0.png">

**Reply by HankMoody089** [reply]
Manage the list is only if you choose "manually"

**Comment by MELASWOJTE** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/97093015/228274708-e41510cc-1d91-4b37-a861-16664dc7249d.png)

**Reply by MELASWOJTE** [reply]
where can i change between manual and autonatic?

**Reply by MELASWOJTE** [reply]
sorry between manual and automatic?
In the Firefly 1.7.1 i only see my SMR tokens, is it ok like this?

**Reply by HankMoody089** [reply]
You are in the Firefly SHIMMER version here, not in the normal Firefly. That's not 1.7.1
Check again

**Reply by HankMoody089** [reply]
<img width="1657" alt="Bildschirmfoto 2023-03-28 um 16 49 33" src="https://user-images.githubusercontent.com/60474745/228277569-9006555c-fa49-43c3-9059-4f735305b34d.png">

You need the left download, Firefly Desktop

**Reply by MELASWOJTE** [reply]
Yes of course. Thank you very much

**Reply by MELASWOJTE** [reply]
it works. Thank you

**Comment by Reclu2** [comment]
hello, you can still stake ASMB?
